src/surfaces/card/card.tsx

```diff
@@ -0,0 +1,24 @@
+import MUICard from '@material-ui/core/Card';
+import React from 'react';
+
+type PropTypes = {
+    raised?: boolean;
+};
+
+function Card(props: PropTypes) {
+    const {
+        raised = false,
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICard
+            elevation={raised && 22}
```

Comment:
NAB: I'd prefer a ternary to keep the underlying type correct.
Maybe `elevation={raised ? 22 : undefined}` or whatever an appropriate default for `elevation` is.
Boolean expressions as syntactic sugar for defensive checks is JavaScript silliness IMO; we're using TypeScript and we're beyond that :-P

Reply:
Fixed.

Reply:
Nice! Thanks!